tests/mod.test.ts

```diff
@@ -0,0 +1,245 @@
+import { assert, assertArrayContains, assertEquals, delay } from "../deps.ts";
+import {
+  botID,
+
+  cache,
+  Channel,
+  createClient,
+
+  createGuildChannel,
+  createGuildRole,
+
+  createServer,
+  deleteChannel,
+
+  deleteRole,
+
+  deleteServer,
+  editRole,
+
+  getMessage,
+
+  Guild,
+  Intents,
+
+  OverwriteType,
+  Role,
+
+  sendMessage,
+} from "../mod.ts";
+import { editChannel } from "../src/handlers/channel.ts";
+import { getChannel } from "../src/handlers/guild.ts";
+
+const token = Deno.env.get("DISCORD_TOKEN");
+if (!token) throw "Token is not provided";
+
+createClient({
+  token,
+  intents: [Intents.GUILD_MESSAGES, Intents.GUILDS],
+});
+
+// Default options for all test cases
+const testOptions = {
+  sanitizeOps: false,
+  sanitizeResources: false,
+};
+
+Deno.test({
+  name: "connect to the gateway",
+  fn: async () => {
+    // Delay the execution by 15 seconds (15000 ms)
+    await delay(15000);
+
+    // Check whether botID is nil or not
+    assert(botID);
+  },
+  ...testOptions,
+});
+
+let guildID: string;
```

Comment:
For long term wise we should move all these vars into 1 placeholder at the top like

```ts
const data = {
  guildID: "",
   roleID: ""
};
```

Then we can assign and delete by data.guildID = x

Reply:
It would save us from having to createa ton of vars